yarn-project/end-to-end/src/e2e_nft.test.ts

```diff
@@ -0,0 +1,156 @@
+import { type AccountWallet, AztecAddress, BatchCall, ExtendedNote, Fr, Note } from '@aztec/aztec.js';
+import { deriveStorageSlotInMap } from '@aztec/circuits.js/hash';
+import { pedersenHash } from '@aztec/foundation/crypto';
+import { NFTContract } from '@aztec/noir-contracts.js';
+
+import { jest } from '@jest/globals';
+
+import { setup } from './fixtures/utils.js';
+
+const TIMEOUT = 120_000;
+
+// This is a very simple test checking only the happy path. More complete tests of the NFT are implemented with TXE.
+// This test is only kept around to check that public data writes are squashed as expected.
```

Comment:
This will also serve as a smoke test for partial note delivery right?

Reply:
Grego told me that TXE spins up its own NoteProcessor so theoretically it should not be needed but I would say that yes, it's good to have an e2e flow for it.